src/util/base58.rs

```diff
@@ -13,23 +13,22 @@ use crate::hashes::{sha256d, Hash};
 use crate::prelude::*;
 use crate::util::endian;
 
-/// An error that might occur during base58 decoding
+/// An error that might occur during base58 decoding.
 #[derive(Debug, PartialEq, Eq, PartialOrd, Ord, Hash, Clone)]
 #[non_exhaustive]
 pub enum Error {
-    /// Invalid character encountered
+    /// Invalid character encountered.
     BadByte(u8),
-    /// Checksum was not correct (expected, actual)
+    /// Checksum was not correct (expected, actual).
     BadChecksum(u32, u32),
```

Comment:
Would prefer this to be `{ expected: u32, got: u32 }` but not in this PR.

Reply:
Yes, I'll keep in mind.